seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/parse/MultipleTableJobConfigParser.java

```diff
@@ -476,8 +453,10 @@ public static SeaTunnelDataType<?> getProducedType(Action action) {
                         .getProducedCatalogTable()
                         .getSeaTunnelRowType();
             } catch (UnsupportedOperationException e) {
```

Comment:
I think it should not throw `UnsupportedOperationException` after this change.
```suggestion
```

Reply:
Done

Reply:
> I think it should not throw `UnsupportedOperationException` after this change.

done.